handsontable/src/core.js

```diff
@@ -3325,6 +3325,16 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
     });
   };
 
+  /**
+   * Returns the meta information for the provided column.
+   *
+   * @param {number} column Visual column index.
+   * @returns {object}
+   */
```

Comment:
```suggestion
  /**
   * Returns the meta information for the provided column.
   *
   * @since 14.5
   * @memberof Core#
   * @function getColumnMeta
   * @param {number} column Visual column index.
   * @returns {object}
   */
```